admin-dev/themes/new-theme/scss/components/layout/_content_div.scss

```diff
@@ -28,9 +28,20 @@
     padding-top: 0;
 
     .header-toolbar {
-      padding-left: 0;
-      margin: 0 -#{$grid-gutter-width / 2};
-      margin-bottom: 1rem;
+      display: none;
+    }
+  }
+
+  @at-root .multishop-enabled & {
+
```

Comment:
```suggestion
```

Reply:
These empty lines means there is a parent child relationship.
So debugging would be easier.
Btw these empty lines will be deleted by Sass after compile.

```
.parent {
    color: $white;
    
    .child {
        background: $dark;
        text-align: $align;
    }
}
```

Reply:
It doesn't help the readability if the parent doesn't have any attribute, the tabulation does it by itself, I agree with @sowbiba on this I never did this, but maybe we could autofix it with stylelint